Sources/Testing/Testing.docc/EnablingAndDisabling.md

```diff
@@ -3,26 +3,26 @@
 <!--
 This source file is part of the Swift.org open source project
 
-Copyright (c) 2023 Apple Inc. and the Swift project authors
+Copyright (c) 2023-2024 Apple Inc. and the Swift project authors
 Licensed under Apache License v2.0 with Runtime Library Exception
 
 See https://swift.org/LICENSE.txt for license information
 See https://swift.org/CONTRIBUTORS.txt for Swift project authors
 -->
 
-Conditionally enable or disable individual tests before they run.
+Enable or disable individual tests before they run.
```

Comment:
"Conditionally" is important here, because this article discusses how to enable/disable tests based on a runtime condition. We could rephrase maybe?